src/Select/SelectInput.d.ts

```diff
@@ -13,8 +13,8 @@ export interface SelectInputProps extends StandardProps<{}, SelectInputClassKey>
   onChange?: (event: React.ChangeEvent<{}>, child: React.ReactNode) => void;
   onFocus?: React.FocusEventHandler<any>;
   readOnly?: boolean;
-  renderValue?: Function;
-  selectRef?: Function;
+  renderValue?: (value: SelectInputProps['value']) => React.ReactNode;
+  selectRef?: (ref: HTMLSelectElement | HTMLInputElement) => void;
```

Comment:
It looks like when `!native`, what actually gets passed to `selectRef` is

```ts
{
  node: HTMLInputElement;
  value: SelectInputProps['value'];
}
```

Reply:
Yeah, it's something I missed. Just updated it here.